puppet-lint-check_unsafe_interpolations.gemspec

```diff
@@ -0,0 +1,29 @@
+require_relative 'lib/puppet/lint/check_unsafe_interpolations/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "puppet-lint-check_unsafe_interpolations"
+  spec.version       = Puppet::Lint::CheckUnsafeInterpolations::VERSION
+  spec.authors       = 'Vox Pupuli'
+  spec.email         = 'voxpupuli@groups.io'
+  spec.files         = Dir[
+    'README.md',
+    'LICENSE',
+    'lib/**/*',
+    'spec/**/*',
+  ]
+  spec.test_files  = Dir['spec/**/*']
+  spec.summary       = 'A puppet-lint plugin to check for unsafe interpolations.'
+  spec.description   = <<-EOF
+    A puppet-lint plugin to check for unsafe interpolations.
+  EOF
+  spec.homepage      = 'https://github.com/voxpupuli/puppet-lint-check_unsafe_interpolations'
```

Comment:
This url should be "https://github.com/puppetlabs/puppet-lint-check_unsafe_interpolations"